src/components/List/ListItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { HTMLAttributes } from 'react'
+
+import { GenericComponentProps } from '../../interfaces/GenericComponentProps'
+import { ColorTheme } from '../../types/ColorTheme'
+import { ComponentSizeSmallMediumLarge } from '../../types/ComponentSize'
+import { IconType } from '../../types/IconType'
+import { PhoenixIconsColored } from '../../types/PhoenixIcons'
+import { TextColor } from '../../types/TextColor'
+import { MarginProps } from '../common/Spacing/MarginProps'
+import ListItemIcon from './ListItemIcon'
+import { StyledListItem } from './ListStyles'
+
+export interface ListCommonProps {
+	size?: ComponentSizeSmallMediumLarge | string | number
+	color?: TextColor
+	icon?: IconType | PhoenixIconsColored | null
+	colorTheme?: ColorTheme
+	bulletColor?: TextColor
+}
```

Comment:
I'd move this to a separate file, because it doesn't really belong to either component.

Reply:
Good idea. I moved it to a separate file and also moved there `commonPropsKeys` which was in a separate file because it logically belongs together.